tests/snuba/api/endpoints/test_organization_events_stats.py

```diff
@@ -64,17 +64,17 @@ def test_simple(self):
             self.url,
             data={
                 "start": iso_format(self.day_ago),
-                "end": iso_format(self.day_ago + timedelta(hours=1, minutes=59)),
+                "end": iso_format(self.day_ago + timedelta(hours=2)),
                 "interval": "1h",
             },
             format="json",
         )
 
         assert response.status_code == 200, response.content
         assert [attrs for time, attrs in response.data["data"]] == [
-            [],
             [{"count": 1}],
             [{"count": 2}],
+            [{"count": 0}],
```

Comment:
Why are there 3 entries here?

Reply:
@billyvg Good question.

I updated this test to ensure returned data points are only within the requested time range. For example:

If start= 12 PM, end = 2 PM, and interval = 1 hour, then we should expect 3 data points. These points refer to 12 PM, 1 PM, and 2 PM buckets.